[2022-08-23 16:14:20,674] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.step_01 scheduled__2022-01-05T12:00:00+00:00 [queued]>
[2022-08-23 16:14:20,684] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.step_01 scheduled__2022-01-05T12:00:00+00:00 [queued]>
[2022-08-23 16:14:20,685] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 16:14:20,685] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-23 16:14:20,689] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 16:14:20,705] {taskinstance.py:1397} INFO - Executing <Task(BashOperator): step_01> on 2022-01-05 12:00:00+00:00
[2022-08-23 16:14:20,712] {standard_task_runner.py:52} INFO - Started process 599 to run task
[2022-08-23 16:14:20,718] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'raizen_lab_01', 'step_01', 'scheduled__2022-01-05T12:00:00+00:00', '--job-id', '8096', '--raw', '--subdir', 'DAGS_FOLDER/raizen_lab_01.py', '--cfg-path', '/tmp/tmpdnzcnbz2', '--error-file', '/tmp/tmpzh14m5r6']
[2022-08-23 16:14:20,720] {standard_task_runner.py:80} INFO - Job 8096: Subtask step_01
[2022-08-23 16:14:20,808] {task_command.py:371} INFO - Running <TaskInstance: raizen_lab_01.step_01 scheduled__2022-01-05T12:00:00+00:00 [running]> on host 8790b68fb5bc
[2022-08-23 16:14:20,901] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=raizen_lab_01
AIRFLOW_CTX_TASK_ID=step_01
AIRFLOW_CTX_EXECUTION_DATE=2022-01-05T12:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-01-05T12:00:00+00:00
[2022-08-23 16:14:20,903] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-08-23 16:14:20,904] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo "hello"']
[2022-08-23 16:14:20,919] {subprocess.py:85} INFO - Output:
[2022-08-23 16:14:20,921] {subprocess.py:92} INFO - hello
[2022-08-23 16:14:20,922] {subprocess.py:96} INFO - Command exited with return code 0
[2022-08-23 16:14:20,957] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=raizen_lab_01, task_id=step_01, execution_date=20220105T120000, start_date=20220823T161420, end_date=20220823T161420
[2022-08-23 16:14:21,009] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-23 16:14:21,049] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
